packages/rockets-next-web/src/components/App/App.tsx

```diff
@@ -14,13 +15,18 @@ const App = ({ children }: { children: React.ReactNode }) => {
   const { darkMode } = (useContext?.(ThemeContext) as ThemeContextType) || {};
 
   return (
-    <ThemeProvider theme={darkMode ? themeDark : themeLight}>
-      <AuthProvider>
-        <html lang="en">
-          <body className={inter.className}>{children}</body>
-        </html>
-      </AuthProvider>
-    </ThemeProvider>
+    <ClientProvider
+      baseUrl={process.env.NEXT_PUBLIC_API_URL}
+      onRefreshTokenError={() => {}}
```

Comment:
Lets Configure the same alert that we did on MSW and ECN

Reply:
From what I've seen on those two projects, they use `react-toastify` for alerts. Should I install it on this project or is it present in the `rockets-react` repo?

Reply:
Yes!

Reply:
For this, I followed the same structure present in [this ECN file](https://gitlab.com/concepta/ecn/ecn-portal/-/blob/develop/packages/web/src/app/layout.tsx?ref_type=heads). It doesn't show a toast per se when `onRefreshTokenError` is called, but redirects the user to the login page.